nipype/interfaces/ants/segmentation.py

```diff
@@ -384,6 +387,23 @@ def _list_outputs(self):
                 self._gen_filename('bias_image'))
         return outputs
 
+    def _run_interface(self, runtime, correct_return_codes=(0,)):
+        runtime = super(N4BiasFieldCorrection, self)._run_interface(
+            runtime, correct_return_codes)
+
+        if self.inputs.copy_header:
+            import nibabel as nb
+            refnii = nb.load(self.inputs.input_image)
+            hdr = refnii.header.copy()
+            out_file = self._gen_filename('output_image')
+            nii = nb.load(out_file)
+            hdr.set_data_dtype(nii.header.get_data_dtype())
+            nb.Nifti1Image(nii.get_data(), refnii.affine, hdr).to_filename(
+                out_file)
```

Comment:
`N4BiasFieldCorrection` virtually accepts any [of the file formats supported by ITK](https://itk.org/Wiki/ITK/File_Formats). So I guess it is worth checking the image type (Analyze or Nifti) before copying the header. @satra any thoughts?